packaging/windows/setup_windows.iss

```diff
@@ -103,45 +107,45 @@ begin
   exit;
 end;
 
-function IsKeybaseRunning : Boolean;
-var
-  WMIService: Variant;
-  WbemLocator: Variant;
-  WbemObjectSet: Variant;
-
-begin
-  WbemLocator := CreateOleObject('WbemScripting.SWbemLocator');
-  WMIService := WbemLocator.ConnectServer('localhost', 'root\CIMV2');
-  WbemObjectSet := WMIService.ExecQuery(ExpandConstant('SELECT * FROM Win32_Process Where Name="{#MyExeName}"'));
-  
-  // Fairly simple check just to see if a process is running named Keybase.exe
-  // No point in trying to stop it otherwise (it will hang).
-  Result := not VarIsNull(WbemObjectSet) and (WbemObjectSet.Count > 0);
-end;
- 
-procedure StopKeybaseService();
-var
-  ResultCode: Integer;
-  CommandName: string;
-  i: Integer;
-
-begin
-  if IsKeybaseRunning() then
-  begin
-    // Launch Keybase ctl stop and wait for it to terminate
-    CommandName := ExpandConstant('{app}\{#MyExeName}');
-    Exec(CommandName, 'ctl stop', '', SW_SHOW,
-      ewWaitUntilTerminated, ResultCode);
-    Sleep(500);
-  end;
-  for i := 1 to 5 do 
-  begin
-    if IsKeybaseRunning() then
-      Sleep(500)
-    else
-      break;
-  end;
-end;
+// function IsKeybaseRunning : Boolean;
```

Comment:
Instead of commenting out tens of lines of code, I think it's better to just remove them.  Git stores the history, so you can always go back to it there.